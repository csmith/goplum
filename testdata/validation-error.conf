# http.get requires a url field
check http.get "test" {}
