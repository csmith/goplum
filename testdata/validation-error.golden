"error configuring check test: missing required argument: url"